packages/dev/core/src/Gizmos/axisScaleGizmo.ts

```diff
@@ -151,7 +170,7 @@ export class AxisScaleGizmo extends Gizmo implements IAxisScaleGizmo {
         this._rootMesh.addBehavior(this.dragBehavior);
 
         let currentSnapDragDistance = 0;
-        const tmpVector = new Vector3();
+

```

Comment:
For future readability - you could have done this:

```javascript
const tmpVector = TmpVectors.Vector3[0];
```

and then change nothing else

Reply:
True! I'll keep that in mind in case it happens again

Reply:
feel free to resolve and merge :-)